scenarios/action_recognition/README.md

```diff
@@ -1,66 +1,111 @@
+The following example notebooks are provided:
 
 | Notebook | Description |
 | --- | --- |
-| [00_webcam](00_webcam.ipynb) | A real-time inference example on Webcam stream |
-| [01_training_introduction](01_training_introduction.ipynb) | An example of training R(2+1)D model on HMDB-51 dataset |
-| [02_video_transformation](02_video_transformation.ipynb) | Examples of video transformations | 
+| [00_webcam](00_webcam.ipynb) | Real-time inference example on Webcam input. |
+| [01_training_introduction](01_training_introduction.ipynb) | Fine-tuning on the HMDB-51 dataset. |
+| [02_video_transformation](02_video_transformation.ipynb) | Examples of video transformations. |
 
-Specifically, we use the model pre-trained on 65 million social media videos (IG) presented in "[Large-scale weakly-supervised pre-training for video action recognition (2019)](https://arxiv.org/abs/1905.00561)" paper.
+Furthermore, tools for data annotation are located in the [video_annotation](./video_annotation) subfolder.
 
-*Note: The official pretrained model weights can be found from [https://github.com/facebookresearch/vmz](https://github.com/facebookresearch/vmz) which are based on caffe2.
-In this repository, we use PyTorch-converted weights from [https://github.com/moabitcoin/ig65m-pytorch](https://github.com/moabitcoin/ig65m-pytorch).*
 
+## Technology
 
-## Prerequisite
-* Linux machine - We strongly recommend to use GPU machine to run the scripts and notebooks in this project smoothly (preferably [Azure NCsV3 series VMs](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-gpu#ncv3-series)).
-* To use GPUs, **CUDA toolkit v10.1** is required. Details about the CUDA installation can be found [here](https://developer.nvidia.com/cuda-downloads). Once the installation is completed, you may need to reboot the VM.
+Action recognition is an active field of research, with large number of approaches being published every year. One of the approaches which stands out is the  **R(2+1)D** model which is described in the 2019 paper "[Large-scale weakly-supervised pre-training for video action recognition](https://arxiv.org/abs/1905.00561)".
 
+R(2+1)D is highly accurate and at the same time significantly faster than other approaches:
+- Its accuracy comes in large parts from an extra pre-training step which uses 65 million automatically annotated video clips.
+- Its speed comes from simply using video frames as input. Many other state-of-the-art methods require optical flow fields to be pre-computed which is computationally expensive (see the "Inference speed" section below).
 
-## Installation
-1. Setup conda environment
-`conda env create -f environment.yml`
+We base our implementation on this [github](https://github.com/moabitcoin/ig65m-pytorch) repository, with added functionality to make training and evaluating custom models more user-friendly. We were able to re-produce the published accuracy for the HMDB-51 dataset as can be seen in this table:
```

Comment:
Again, we don't borrow any codes from that repo. We use the pretrained model weight from the repo (which the author uploaded to torchhub).

Reply:
Thanks for the clarification. I'll have a few related questions to this once we meet.